client/my-sites/domains/domain-management/dns/dns-record-data.jsx

```diff
@@ -0,0 +1,116 @@
+import { Button, Gridicon } from '@automattic/components';
+import { localize } from 'i18n-calypso';
+import PropTypes from 'prop-types';
+import { Component } from 'react';
+import DnsRecordsListItem from './dns-records-list-item';
+
+class DnsRecordData extends Component {
+	static propTypes = {
+		actions: PropTypes.array.isRequired,
+		dnsRecord: PropTypes.object.isRequired,
+		enabled: PropTypes.bool,
+		onDeleteDns: PropTypes.func.isRequired,
+		selectedDomainName: PropTypes.string.isRequired,
+	};
+
+	static defaultProps = {
+		enabled: true,
+	};
+
+	handledBy() {
+		const { dnsRecord, translate } = this.props;
+		const { type, aux, port, service, weight, protocol } = dnsRecord;
+		const data = this.trimDot( dnsRecord.data );
+		const target = this.trimDot( dnsRecord.target );
+
+		// TODO: Remove this once we stop displaying the protected records
+		if ( dnsRecord.protected_field ) {
+			if ( 'MX' === type ) {
+				return translate( 'Mail handled by WordPress.com email forwarding' );
+			}
+
+			return translate( 'Handled by WordPress.com' );
+		}
+
+		switch ( type ) {
+			case 'MX':
+				return translate( '%(data)s with priority %(aux)s', {
+					args: {
+						data,
+						aux,
+					},
+				} );
+
+			case 'SRV':
+				return translate(
+					'%(service)s (%(protocol)s) on target %(target)s:%(port)s, ' +
```

Comment:
:information_source: String reuse speeds up translation and improves consistency. The following string might make a good alternative and has already been translated **22** times:
`translate( 'Service %(service)s (%(protocol)s) on target %(target)s:%(port)s, with priority %(aux)s and weight %(weight)s' )` *ES Score: 9*